client/src/components/DisplayTable.js

```diff
@@ -114,7 +114,7 @@ export class DisplayTable extends React.Component {
                   column_keys, 
                   col => (
                     <td key={col}>
-                      {col_data[col]}
+                      {`${col_data[col]}`}
```

Comment:
I had some data being stored as true or false rather than strings and this was a hack to get it to show up (rather than just be blank). I'll add a TODO to change it since that will have to be changed either way to accommodate translation.